packages/element/src/react.js

```diff
@@ -40,6 +40,12 @@ import { isString } from 'lodash';
  * @typedef {import('react').ComponentType} WPComponent
  */
 
+/**
+ * Object containing a React node.
+ *
+ * @typedef {import('react').ReactNode} WPNode
+ */
+
```

Comment:
I'd vote for dropping the aliases and using the react types directly. There's no good way to re-export TS types via JavaScript and this isn't going to be sustainable.